features/step_definitions/session-storage.js

```diff
@@ -0,0 +1,14 @@
+let session;
+
+function create(_session) {
```

Comment:
Later we get make the session a property using Proxy. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy